packages/sentry/src/sentry.ts

```diff
@@ -74,24 +79,36 @@ export const redactUnsafeEventFields = (event: Event) => {
 
 export const boot = () => {
   if (window.app.trackingSentry && window.app.trackingSentry !== 'null') {
+    const integrations: Integration[] = [
+      new Sentry.Integrations.GlobalHandlers({
+        onunhandledrejection: false,
+        onerror: false,
+      }),
+    ];
+
+    if (canUseDOM) {
+      // Require the browser history on runtime. This allows the package to be used on SSR.
+      const {
+        default: browserHistory,
+      } = require('@commercetools-frontend/browser-history');
```

Comment:
Just wondering is we shouldn't use the ESM `import` here 🤔

Reply:
I get it, but the code within this if statement block runs in the browser. Can CJS `required` be used there?

Reply:
Sure, but I think that `require` is transformed in compilation time the same as most imports when the build target is, for instance, ES5.

Reply:
I'm not sure, but I think the failing E2E tests are impacted exactly by this. The `playground` app crashes:
 
<img width="629" alt="Screenshot 2023-03-07 at 14 16 03" src="https://user-images.githubusercontent.com/49066275/223433147-a54bee49-c0cd-4d13-ab66-4fb19c609c8e.png">

Reply:
You're right, `require` doesn't work with Vite because it expects ESM.

I have to rethink the approach then 🤔